[REL-3012] Explain how to use PROJECT ID from env vars (#8768)
circleci/circleci-docs: jekyll/_cci2/release/configure-your-kubernetes-components.adoc

+
[NOTE]
====

> Find or use the project ID in two ways:

Reviewer: ```suggestion
There are two ways to specify your **project ID**, as follows:
```

> * Use it within a CircleCI Pipeline as `$CIRCLE_PROJECT_ID` from the built-in environment variables link:https://circleci.com/docs/variables/#built-in-environment-variables[CircleCI Variables]. This way, you can use tools like `envsubst` to substitute the placeholder for the real value.

Reviewer: ```suggestion
* Use the `$CIRCLE_PROJECT_ID` xref:variables#built-in-environment-variables[built-in environment variable] in a CircleCI pipeline. This way, you can use tools like `envsubst` to substitute the placeholder for the real value.
```
